src/ServiceManagement/RemoteApp/Commands.RemoteApp/packages.config

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Authentication" version="1.0.10-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Extensions" version="0.13.1-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.RemoteApp" version="1.0.7" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.14.1-preview" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
+  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
+  <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net45" />
+  <package id="Microsoft.Data.Edm" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.Data.OData" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.Data.Services.Client" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="3.0.110281957-alpha" targetFramework="net45" />
+  <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net45" />
+  <package id="Microsoft.WindowsAzure.ConfigurationManager" version="1.8.0.0" targetFramework="net45" />
+  <package id="Microsoft.WindowsAzure.Management" version="4.1.0" targetFramework="net45" />
+  <package id="Microsoft.WindowsAzure.Management.Compute" version="9.1.0" targetFramework="net45" />
```

Comment:
Updated to use the one used by other extensions for now, otherwise RemoteApp will be the only project depending on the proposed version (9.2) of this library